tools/structure/src/config/properties.json

```diff
@@ -2,7 +2,7 @@
   "schemaVersion": "0.0.1",
   "displayName": "Structure Tool",
   "functionName": "structure_tool",
-  "toolVersion": "0.0.44",
+  "toolVersion": "0.0.45",
```

Comment:
@jagadeeswaran-zipstack Did we take build for this ? Couldn't locate this in action runs.
Please verify the versions

Reply:
https://github.com/Zipstack/unstract/pull/798#discussion_r1802957975